docs/operational_guide/placement_configuration.md

```diff
@@ -253,3 +253,65 @@ initialCluster:
    `/health`) endpoint before continuing to the next.
 
 6. Follow the steps from `Replacing a Seed Node` to replace `host3` with `host4` in the M3DB placement.
+
+#### Setting a new placement (Not Recommended)
+
+This endpoint is unsafe since it creates a brand new placement and therefore should be used with extreme caution.
+Some use cases for using this endpoint include:
+
+- Changing IP addresses of seed nodes
```

Comment:
Can just say, nodes - mentioning seed nodes here might confuse more than help understand how the endpoint is used.